.github/workflows/build.yml

```diff
@@ -103,14 +103,20 @@ jobs:
       run: echo "sha_short=$(git rev-parse --short HEAD)" >> $GITHUB_OUTPUT
       working-directory: ${{ matrix.vcpkg_path }}
 
+    # python3-distutils is required for fontconfig but it has been removed in Python 3.12
+    - name: Setup Python 3.11
+      uses: actions/setup-python@v4
+      with:
+          python-version: '3.11'
+
     - name: Bootstrap vcpkg
       run: ${{ matrix.vcpkg_bootstrap }}
       working-directory: ${{ matrix.vcpkg_path }}
 
     - name: "[macOS] Bootstrap vcpkg"
       if: runner.os == 'macOS'
       run: |
-          brew update && brew install nasm automake pkg-config
+          brew update && brew install nasm
```

Comment:
`nasm `can also be aquired using VCPKG itself: https://learn.microsoft.com/de-de/vcpkg/maintainers/functions/vcpkg_find_acquire_program
For which port is it needed? Maybe the port has a missing dependency to nasm?

Reply:
Unfortunately we have not documented it. 
It has been original introduced as yasm and then upgrated to nasm: 
https://github.com/mixxxdj/vcpkg/commit/523aef6098c0954a3cd1376c11c311cf6bedd49c
I will check what happens without it.

Reply:
It is Ffmpeg: 
```
CMake Error at scripts/cmake/vcpkg_find_acquire_program.cmake:619 (message):
  Could not find nasm.  Please install it via your package manager:

      brew install nasm
Call Stack (most recent call first):
  ports/ffmpeg/portfile.cmake:31 (vcpkg_find_acquire_program)
  scripts/ports.cmake:147 (include)


error: building ffmpeg:x64-osx-min1012 failed with: BUILD_FAILED
```

I will add a comment. I think we should apply the port change via upstream if we want to fix it. IMHO the effort of moving ffmpeg to the overly is not worth it. I have no interest to pick up this task.

Reply:
My tip to use vcpkg_find_acquire_program was not a good one, because this only downloads the package on Windows. On macOS it only prints to use Homebrew.
I read upstream is working on making these build tools available as full featured ports - but this seems to be a long term goal.